runtime/doc/options.txt

```diff
@@ -1779,9 +1779,6 @@ A jump table for the options with a short description can be found at |Q_op|.
 								*cpo-I*
 		I	When moving the cursor up or down just after inserting
 			indent for 'autoindent', do not delete the indent.
-								*cpo-j*
-		j	When joining lines, only add two spaces after a '.',
```

Comment:
Good catch, I'll just remove the line

        When 'cpoptions' includes the 'j' flag, only do this after a '.'.